scripts/image_converter/requirements.md

```diff
@@ -0,0 +1,5 @@
+# Sketch image converter
```

Comment:
This file should be named readme.md
Make a seperate file called requirements.txt for requirements.

Reply:
done